kroxylicious-operator/src/test/java/io/kroxylicious/kubernetes/operator/VirtualKafkaClusterReconcilerTest.java

```diff
@@ -186,7 +187,7 @@ static List<Arguments> shouldSetResolvedRefsToTrueOrFalse() {
                                 .withStatus(Condition.Status.FALSE).endCondition().endStatus().build()),
                         (Consumer<ConditionAssert>) ca -> ca.isResolvedRefsFalse(
                                 VirtualKafkaClusterReconciler.TRANSITIVELY_REFERENCED_RESOURCES_NOT_FOUND,
-                                "spec.filterRefs references kafkaprotocolfilter.filter.kroxylicious.io/my-filter")));
+                                "spec.filterRefs references kafkaprotocolfilter.filter.kroxylicious.io/my-filter in namespace 'my-namespace'")));
```

Comment:
This enough unresolvable references we're gonna end up with the really big condition message with a lot of repeated substrings! We might eventually want to limit the number of things we report about, but it's definitely not something to worry about right now.